sklearn/utils/fixes.py

```diff
@@ -21,6 +21,12 @@
 
 from .deprecation import deprecated
 
+try:
+    import pkg_resources
+    parse_version = pkg_resources.parse_version
```

Comment:
Looks like mypy is complaining about the fact that `parse_version` does not have the same signature in both cases. Maybe @rth knows how to tell mypy to not check this? Otherwise googling is definitely an option.

https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=18541&view=logs&j=32e2e1bb-a28f-5b18-6cfc-3f01273f5609&t=8a54543f-0728-5134-6642-bedd98e03dd0
```
sklearn/utils/fixes.py:28: error: Incompatible types in assignment (expression has type "Type[LooseVersion]", variable has type "Callable[[str], Tuple[str, ...]]")
```

Reply:
Yeah, you can just add
```
# type: ignore
```
at the end of each line. There is no easy fix to make those signatures match. https://mypy.readthedocs.io/en/stable/common_issues.html#spurious-errors-and-locally-silencing-the-checker

Reply:
@rth Thank you for your advice!
I tried to add it at the end of appropriate lines.